source/includes/_platform-payments.md

```diff
@@ -413,11 +414,56 @@ Wallet Details
 | balance.amount              | number | The amount.                                      |
 | balance.currency            | string | The ISO 4217 three character codes eg 'GBP'      |
 
+## `DELETE /wallets/{id} 🚧`
```

Comment:
Not sure we'd want to support this? Wonder if this was documented but we removed since it wasn't a feature we wanted to support (at least right now). Unless we have already thought about redirects / closing the Modulr account and if a wallet has a balance, etc.

Reply:
The endpoint exist in payments already.
AT for it:

https://github.com/Goji-P2P/payments-application/blob/7d6463d250af26613789a913a049b1e1e5e250cc/acceptance-tests/src/test/java/com/gojiholdings/payments/acceptancetests/component/wallet/WalletCreationAcceptanceTest.java#L58

But TBH I'm not 100% sure that endpoint is working correctly though. I've never used it.

Reply:
I can remove it from the docs for now.